build/dockerfiles/entrypoint.sh

```diff
@@ -16,13 +16,20 @@
 # By default, this script will operate on the `/var/www/html/devfiles` directory.
 # This can be overridden by the environment variable $DEVFILES_DIR
 #
+# In addition, this script will perform the necessary set up for the offline
+# devfile registry, replacing placeholders in all devfiles based off environment
+# variable
+#     CHE_DEVFILE_HTTPS_ENDPOINT
```

Comment:
So I assume that in airgap mode, from the operator, we would also have to add this variable in the new devfile registry config map, based on the external URL of the devfile registry ?

@l0rd do you confirm ?

cc @tomgeorge

Reply:
Yes, that's accurate; we need the environment variable to be able to rewrite devfiles to point at exposed endpoints. Without the environment variable we do try to use k8s-provisioned env vars to determine service IP and route, but I haven't tested this much and it may be unreliable.

Reply:
@davidfestal yes that's the weakness of this solution. There is no reliable way to figure the value automatically.

Reply:
In particular, without the env var set, we try to go off of 
```
URL="http://${CHE_DEVFILE_REGISTRY_SERVICE_HOST}:${CHE_DEVFILE_REGISTRY_SERVICE_PORT}"
```

which depends on the devfile registry server being called `che-devfile-registry`, and may not work in some scenarios (I don't know the internals of networking on OpenShift).